Makefile

```diff
@@ -1,13 +1,24 @@
 root_dir := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+word-dot = $(word $2,$(subst ., ,$1))
 
 TARGET = pg_query
 ARLIB = lib$(TARGET).a
+SOLIB = lib$(TARGET).so
 PGDIR = $(root_dir)/tmp/postgres
 PGDIRBZ2 = $(root_dir)/tmp/postgres.tar.bz2
 
 PG_VERSION = 13.2
+PG_VERSION_MAJOR = $(call word-dot,$(PG_VERSION),1)
 PROTOC_VERSION = 3.14.0
 
+VERSION = 2.0.4
+VERSION_MAJOR = $(call word-dot,$(VERSION),1)
+VERSION_MINOR = $(call word-dot,$(VERSION),2)
+VERSION_PATCH = $(call word-dot,$(VERSION),3)
+
+SONAME = $(SOLIB).$(shell printf '%02d%02d' $(PG_VERSION_MAJOR) $(VERSION_MAJOR))
```

Comment:
I think we should include the minor version into the SONAME, since we may have ABI breaks with minor releases as well (but it seems reasonable to avoid them with patch releases).

Reply:
Resolved by ade04b03f1e3955558d9489a0e1b5604c0210802.